examples/vault-cluster-auto-unseal/README.md

```diff
@@ -0,0 +1,46 @@
+# Vault Cluster Auto Unseal Example 
+
+This example deploys a publicly accessible [Vault](https://www.vaultproject.io/) cluster in [GCP](https://cloud.google.com/)
+fronted by a Regional External Load Balancer using the [vault-cluster](https://github.com/hashicorp/terraform-google-vault/tree/master/modules/vault-cluster) and [vault-lb-fr](
+/modules/vault-lb-fr) modules. For an example of a private Vault cluster that is accessible only from inside the Google
+Cloud VPC, see [vault-cluster-private](https://github.com/hashicorp/terraform-google-vault/tree/master/examples/vault-cluster-private). **Deploying Vault in a publicly accessible way
+should be avoided if possible due to the increased security exposure. However, it may be unavoidable, if, for example,
+Vault is your system of record for identity.**. 
+
+The Vault cluster uses [Consul](https://www.consul.io/) as a storage backend, so this example also deploys a separate
+Consul server cluster using the [consul-cluster module](
+https://github.com/hashicorp/terraform-google-consul/tree/master/modules/consul-cluster) from the Consul GCP Module.
+
+You will need to create a [Google Image](https://cloud.google.com/compute/docs/images) that has Vault and Consul
+installed, which you can do using the [vault-consul-image example](https://github.com/hashicorp/terraform-google-vault/tree/master/examples/vault-consul-image)).  
+
+Note that a Google Load Balancer requires a Health Check to confirm that the Vault nodes are healthy, but at this time,
+Google Cloud only supports [associating HTTP Health Checks with a Target Pool](
+https://github.com/terraform-providers/terraform-provider-google/issues/18), not HTTPS Health Checks. The recommended
+workaround is to run a separate proxy server that listens over HTTP and forwards requests to the HTTPS Vault endpoint.
+We accomplish this by using the [run-nginx](https://github.com/hashicorp/terraform-google-vault/tree/master/modules/run-nginx) module to run the web server. 
```

Comment:
This is unrelated to this module, but would appreciate your input on whether there's some other way to ditch Nginx. Someone tried to do it in #16, but ultimately that didn't work.

Reply:
The correct endpoint would be `/v1/sys/health?standbyok=true` over HTTPS as used in that PR, but I'd have to dig further into the specifics and see if I can get it working.

Reply:
Ok, not a priority for now but would be nice to fix at some point.